### PR TITLE
Hotkey for dispatcher

### PR DESCRIPTION
**Pull request by StephanSteinert**
Added a hotkey for quickly enabling/disabling the dispatcher.

### CODE REVIEW

File: locale/de/base.cfg [hunk 1]
```diff
@@ -79,4 +79,10 @@ train-left=[LTN] Zug __1__ hat __2__ verlassen.
...
 requester_unscheduled_cargo=[LTN] Zug __1__ hat Anforderungshaltestelle __2__ mit ungeplanter Fracht erreicht.
-requester_left_over_cargo=[LTN] Zug __1__ hat Anforderungshaltestelle __2__ mit verbleibender Fracht verlassen.
\ No newline at end of file
+requester_left_over_cargo=[LTN] Zug __1__ hat Anforderungshaltestelle __2__ mit verbleibender Fracht verlassen.
+
+dispatcher-enabled=[LTN] Disponent aktiviert von __1__.
+dispatcher-disabled=[LTN] Disponent deaktiviert von __1__.
+
+[controls]
+ltn-toggle-dispatcher=LTN-Dispatcher ein-/ausschalten.
```
Comment (Yousei9) [inline locale/de/base.cfg:88]: Should be "LTN-Disponent ein-/ausschalten." for consistency

**Review by Yousei9 — CHANGES_REQUESTED** [review]
Disabling Dispatcher is common enough to warrant a shortcut?

Please use 2 whitespaces per indent for consistency.

**Comment by Yousei9** [comment]
Should i credit your name from GitHub or the more recognized steinio?

**Comment by StephanSteinert** [comment]
Steinio would be better i think.

**Review by Yousei9 — APPROVED** [review]

**Pull request merged**